src/benchmark/md5.c

```diff
@@ -0,0 +1,10 @@
+#include <openssl/md5.h>
```

Comment:
Comparing to openssl is great but comparing to NSS as well would be even greater 😄 . This will also help you getting used to NSS' "interesting" interfaces. You can check [1] and [2] on how to use NSS for hashing.

[1] https://searchfox.org/nss/source/cmd/digest/digest.c#91
[2] https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/nss_sample_code/NSS_Sample_Code_sample3